js/data.js

```diff
@@ -332,6 +365,17 @@ var data = {
         dozent: ["Dr. Anne Baumgraß", "Prof. Dr. Mathias Weske"],
         nameLV: "Event Processing"
     },
+    eventprocessing: {
```

Comment:
This looks like the same key is used twice, that's not valid.
I suggest naming this eventprocessing3LP.